enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -782,8 +760,13 @@ flowchart LR
 The Infrastructure Operator is generally responsible for installing and configuring the Gateway API provider
```

Comment:
If there's no difference between the admin and edit cluster roles, why do we need both?

Reply:
Also, see https://github.com/openshift/enhancements/pull/1741/files#r2007575022, which we've ignored in this update.

Reply:
The admin and edit roles are already defined in Kubernetes and OpenShift.  This diagram shows how the Gateway API concepts map to the OpenShift concepts.  

Is the question more about why "Application Developer" maps to both "admin" and "edit" instead of just to one or the other?  Now that you mention it, it would seem reasonable to me to map "Application Developer" to just "edit".  In practice, anyone who is an admin also has editor privileges.

Reply:
Right, this is about aggregating our roles to the standard cluster roles. "admin" and "edit" will both exist regardless, so we need to aggregate the appropriate policy rules to both. Check out https://github.com/openshift/cluster-ingress-operator/pull/1206 which implements this and LMKWYT

Reply:
It's just odd that the table has two identical lines then.  No changes needed.